packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -74,6 +73,8 @@ const StyledButton = styled(BaseButton)`
  * @param {boolean} props.dismissOnClick If to call onDismiss when clicking.
  * @param {Function} props.onFocus Focus event handler.
  * @param {boolean} props.forcePadding Force padding to button regardless of mode.
+ * @param {boolean} props.isToggled `true` if a toggled pushbutton, `false` if an untoggled
+ * pushbutton, leave unset if not a pushbutton.
```

Comment:
```suggestion
 * @param {boolean|null} props.isToggled `true` if a toggled pushbutton, `false` if an untoggled
 * pushbutton, leave unset if not a pushbutton.
```
`null` or `undefined`?

Reply:
and `isToggled: PropTypes.bool,` in PropTypes (can't comment there).

Reply:
Hm, an isn't an optional property just typed as the optional type or do we really need to do `|undefined` for each of them?

I will add the prop-type though!

Reply:
Done in 8dbbc317c1.

Reply:
To be JSDoc purists we should probably use:
![image](https://user-images.githubusercontent.com/21036224/154510174-1f14bfac-2651-41be-bc7c-0d3f581e8f1c.png)

For me, this particular case is a little special because if you leave it unset it will result in this 3rd state (while bool only has 2 states).